spec/line/bot/v2/misc_spec.rb

```diff
@@ -536,7 +535,7 @@
       expect(body.sent_messages[0].quote_token).to eq('IStG5h1Tz7b...')
     end
 
-    it 'response - success - using hash (not recommended way)' do
+    it 'response - success - using hash (not recommended way)', rbs_test: :skip do
```

Comment:
skip this only in rbs test because request with only hash is unrecommended(undocumented) way. `rbs_test: :skip` is marker for this skipping.